packages-next/keystone/src/lib/cloud/assets.ts

```diff
@@ -0,0 +1,92 @@
+import { Readable } from 'stream';
+import fetch from 'node-fetch';
+import FormData from 'form-data';
+import { ImageMetadata } from '@keystone-next/types';
+
+const CLOUD_IMAGES_DOMAIN = 'imgix.net';
+const CLOUD_GRAPHQL_API_ENDPOINT = 'http://localhost:4000/api/graphql';
+const CLOUD_REST_API_ENDPOINT = 'http://localhost:4000/api/rest';
```

Comment:
These should point to live deployed versions of these before we merge this and ideally we shouldn't include imgix in the domain, it should go through some domain of ours.